edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Constants.cs

```diff
@@ -111,6 +111,8 @@ public static class Labels
             public const string DesiredStatus = "net.azure-devices.edge.desiredStatus";
             public const string NormalizedCreateOptions = "net.azure-devices.edge.normalizedCreateOptions";
             public const string ConfigurationId = "net.azure-devices.edge.configurationId";
+            public const string CreateOptions = "net.azure-devices.edge.create-options";
+            public const string Env = "net.azure-devices.edge.env";
```

Comment:
Can you pls remind me again why we need Env vars as a label, as opposed to just checking the env vars available in the container? 
(I remember discussing this, but don't remember why we decided we need this).

Reply:
We need to know which vars the customer set in the deployment, not the larger set of vars on the container (including the IOTEDGE_* variables we add, anything docker adds, etc). I think it would be difficult to know exactly which vars came from the deployment.

Reply:
Yes, and we need to track the environment variables that were removed, which becomes tricky. So this looks good. Thanks.